### ISSUE TITLE
CustomRequestLogTest fails on ubuntu 18.04 with java 10.0.2

### ISSUE THREAD

**Issue by joakime** [eclipse/jetty.project#3236]
```
-------------------------------------------------------------------------------
Test set: org.eclipse.jetty.server.handler.CustomRequestLogTest
-------------------------------------------------------------------------------
Tests run: 27, Failures: 1, Errors: 0, Skipped: 3, Time elapsed: 6.218 s <<< FAILURE! - in org.eclipse.jetty.server.handler.CustomRequestLogTest
testLogEnvironmentVar  Time elapsed: 0.013 s  <<< FAILURE!
java.lang.AssertionError: 

Expected: is "EnvironmentVar: null"
     but: was "EnvironmentVar: -"
	at org.eclipse.jetty.server.handler.CustomRequestLogTest.testLogEnvironmentVar(CustomRequestLogTest.java:257)
```

See attached surefire-report for system/environment details.
[TEST-org.eclipse.jetty.server.handler.CustomRequestLogTest.xml.gz](https://github.com/eclipse/jetty.project/files/2725406/TEST-org.eclipse.jetty.server.handler.CustomRequestLogTest.xml.gz)

**Comment by lachlan-roberts** [comment]
@joakime can you get the fix cherry-picked back into 9.4.x and then close this issue

**Comment by joakime** [comment]
Done.
